### PR TITLE
Multi-architecture support #1238

### PR DESCRIPTION
**Pull request by tobiasoort**
#1238

- pushes container basis to quarkus v22.0.0.2 
- implements multi-arch (arm64 and amd64 only since thats whats provided upstream @ quarkus)
- no cleanup of old 'arch' build yet, since I don't know what's dependent upon it.

This PR may be imperfect as I don't yet know how to run a test suite here and am hoping PRs have automated QC actions.

**Release Note**
```release-note
- pushed quarkus version to v22.0.0.2 
- implemented multi-arch for amd64/arm64
```

### CODE REVIEW

File: script/Makefile [hunk 1]
```diff
@@ -408,9 +408,11 @@ images: build kamel-overlay maven-overlay bundle-kamelets
 ifneq (,$(findstring SNAPSHOT,$(RUNTIME_VERSION)))
 	./script/package_maven_artifacts.sh -s "$(STAGING_RUNTIME_REPO)" -d "$(CAMEL_K_RUNTIME_DIR)" $(RUNTIME_VERSION)
 endif
-	@echo "####### Building Camel K operator container image..."
+	@echo "####### Building Camel K operator container image for multi architectures..."
 	mkdir -p build/_maven_output
-	docker build -t $(CUSTOM_IMAGE):$(CUSTOM_VERSION) -f build/Dockerfile .
+	docker buildx rm --all-inactive --force
+	docker buildx create --name builder --use
+	docker buildx build --platform=linux/amd64,linux/arm64 -t $(CUSTOM_IMAGE):$(CUSTOM_VERSION) -f build/Dockerfile .
```
Comment (squakez) [inline script/Makefile:415]: Probably we should have a parameter for the platform so that is customizable by the end used

**Review by squakez — COMMENTED** [review]
I think we should have a parame

**Comment by tobiasoort** [comment]
So I just 'manually' added qemu support. This isn't the best way! The cleanest way is through github actions, as described here: https://github.com/marketplace/actions/docker-setup-buildx

It seems though that the build architecture/setup for this project is a bit 'manual' and I don't want to pull all that refactoring into this (intented to be) simple, small fixup PR.

**Comment by tobiasoort** [comment]
I'm currently not able to test this in my local branch due to a dependency on kind-registry which cannot be resolved. I'm pretty sure thats environment-specific so please start the pipelines here again so we can see how it works here. Thanks!

**Comment by tadayosi** [comment]
@tobiasoort Can you squash the commits into single one?  It's too many for the simple fix.

@phantomjinx What do you think about the fix?

**Comment by squakez** [comment]
We can "squash and merge" instead of "rebase and merge". I think however the parameter setting for the platform is a must before going.

**Comment by tobiasoort** [comment]
Do we want the built arch(s) in the `VERSIONFILE`?

**Comment by tobiasoort** [comment]
I'll work on this PR in my fork and create a new one because this mess is only causing useless notifications for you.

**Pull request closed without merging**